dex-it/src/test/scala/com/wavesplatform/it/sync/MultipleMatcherVersionsTestSuite.scala

```diff
@@ -0,0 +1,49 @@
+package com.wavesplatform.it.sync
+
+import com.typesafe.config.{Config, ConfigFactory}
+import com.wavesplatform.dex.domain.order.OrderType.BUY
+import com.wavesplatform.dex.it.api.HasKafka
+import com.wavesplatform.dex.it.api.responses.dex.OrderStatus
+import com.wavesplatform.dex.it.docker.{DexContainer, WavesNodeContainer}
+import com.wavesplatform.it.MatcherSuiteBase
+import com.wavesplatform.it.tags.DexMultipleVersions
+
+@DexMultipleVersions
+class MultipleMatcherVersionsTestSuite extends MatcherSuiteBase with HasKafka {
+
+  private val dexPrevTag  = Option(System.getenv("DEX_MULTIPLE_VERSIONS_PREVIOUS_TAG")).getOrElse("latest")
+  private val nodePrevTag = Option(System.getenv("NODE_MULTIPLE_VERSIONS_PREVIOUS_TAG")).getOrElse("latest")
+
+  override protected def dexInitialSuiteConfig: Config = ConfigFactory.parseString(s"""waves.dex.price-assets = [ "$UsdId", "WAVES" ]""".stripMargin)
+
+  override protected def kafkaServer: Option[String] = Some(s"$kafkaIp:9092")
+
+  lazy val wavesNode2: WavesNodeContainer = createWavesNode("waves-2", tag = nodePrevTag, netAlias = None)
```

Comment:
MultipleMatcherVersionsTestSuite uses multiple versions of Nodes :stuck_out_tongue_closed_eyes:

Reply:
And Matchers too ;)